internal/controllers/utils/constants.go

```diff
@@ -361,13 +343,7 @@ const (
 	AlertmanagerSA                              = "alertmanager"
 )
 
-// TLS Mount Paths
-const (
-	TLSServerMountPath = "/secrets/tls"
-	TLSClientMountPath = "/secrets/smo/tls"
-	CABundleMountPath  = "/secrets/smo/certs"
-	CABundleFilename   = "ca-bundle.crt"
-)
+// NOTE: TLS Mount Paths (TLSServerMountPath, TLSClientMountPath, CABundleMountPath, CABundleFilename) moved to internal/constants
```

Comment:
Do we really need this comment? One can grep for the constants or ask their friendly AI-based IDE about it...